src/Command/ListenAllCommand.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Queue\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Yiisoft\Yii\Console\ExitCode;
+use Yiisoft\Yii\Queue\Cli\LoopInterface;
+use Yiisoft\Yii\Queue\QueueFactoryInterface;
+
+final class ListenAllCommand extends Command
+{
+    protected static $defaultName = 'queue:listen-all';
+    protected static $defaultDescription = 'Listens the all the given queues and executes messages as they come. ' .
```

Comment:
It can be useful not only for development. What about to merge the features to the origin `listen` command?

Reply:
This approach is very CPU-greedy, its usage in production is counterproductive. While most of message brokers allows consumers to listen to them (e.g., socket listening), so they are in idle state until an incoming message, this approach is just an endless loop fressing your CPU.

Reply:
That's up to developers, but I know the company where it was used in production.

Reply:
Yes, that's up to developers. Those who doesn't know what's they doing should be warned.